scrapers/Kpopping.yml

```diff
@@ -0,0 +1,80 @@
+name: Kpopping
+performerByName:
+  action: scrapeXPath
+  queryURL: https://kpopping.com/profiles/idol/{}
+  scraper: performerScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://kpopping.com/profiles/idol/
+    scraper: performerScraper
+xPathScrapers:
+  performerScraper:
+    common:
+      $summary: //div[contains(@class, "summary-only-one")]
+    performer:
+      Name:
+        selector: $summary/section/h1/text()
+        postProcess:
+          - replace:
+              - regex: " †"
+                with: ""
+      Gender:
+        selector: //ol[@class="breadcrumb"]/li[2]/a/text()
+        postProcess:
+          - map:
+              Girl groups: female
+              Boy groups: male
+      URL: //link[@rel="canonical"]/@href
+      Twitter:
+        selector: //a[contains(@class, "fa-twitter")]/@href
+        postProcess:
+          - replace:
+              - regex: https://twitter.com/(\w+)\?lang=en
+                with: $1
+      Instagram:
+        selector: //a[contains(@class, "fa-instagram")]/@href
+        postProcess:
+          - replace:
+              - regex: https://www.instagram.com/(\w+)/\?hl=en
+                with: $1
+      Birthdate:
+        selector: $summary//div[@class="equal"][contains(., "Birthday:")]/following-sibling::div[1]/a/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      DeathDate:
+        selector: $summary//div[@class="equal"][contains(., "Passed away:")]/following-sibling::div[1]/text()
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Country: $summary//strong[contains(., "Country:")]/following-sibling::p/text()
+      Height:
+        selector: $summary//div[@class="equal"][contains(., "Height:")]/following-sibling::div[1]/text()
+        postProcess:
+          - replace:
+              - regex: '\d''\d"[\s\xA0]*\((\d+)[\s\xA0]*cm\)'
```

Comment:
The regex doesnt work with double digit eg 5''10, a \d+ instead of \d could fix that. Even then it feels a bit too complex
A simpler
```yaml
              - regex: '.*\((\d+)[^\)]*cm\).*'
 ```
should work fine since we are only interested for what is in the ()